### ISSUE TITLE
Improve form structure

### ISSUE THREAD

**Issue by OlaBekkevold** [IMTEL/LabManager#99]
Each form should have an unique label and the structure in the forms of the sideviews should be better grouped togheter as right now they are only linked in code
